frontend/language/src/nb.json

```diff
@@ -1046,7 +1046,7 @@
   "resourceadm.switch_toggle": "Toggle av eller på",
   "right_menu.calculations": "Beregninger",
   "right_menu.content": "Innhold",
-  "right_menu.content_empty": "Ingen komponent valgt...",
+  "right_menu.content_empty": "Ingen side valgt...",
```

Comment:
Vi burde bruke ellipsesymbolet (`…`) i stedet for tre punktumer. Men trenger vi det her i det hele tatt? En ellipse signaliserer at det kommer noe mer, men her er det jo ikke noe mer.